core/storage/job/gae_models.py

```diff
@@ -86,6 +93,17 @@ def is_cancelable(self):
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Gets at most limit jobs with respect to a time after recency_msec.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+            recency_msec: int. A time in milliseconds. After this time a
```

Comment:
Is this millisecs since the Epoch, or the number of millisecs earlier than the current time? It's not clear to me.